cmd/otelcol/config/collector/agent_config.yaml

```diff
@@ -120,7 +123,7 @@ service:
   extensions: [health_check, http_forwarder, zpages]
   pipelines:
     traces:
-      receivers: [jaeger, otlp, sapm, zipkin]
+      receivers: [jaeger, otlp, sapm, smartagent/signalfx-forwarder, zipkin]
```

Comment:
I believe we should add this to metrics receivers below as well -- otherwise metric client libraries will not work by default

Reply:
Thanks for spotting this.  Updating docs as well.